tools/EMLassemblyline/eal_table_template.xml

```diff
@@ -22,8 +21,8 @@
          <param name="inputdata" type="data_collection"  collection_type="list"  label="Upload all data files wich are data table" help= "Input a data collection."/>
     </inputs>
     <outputs>
-       <collection name="templates" type="list" label="Templates for data table">
-             <discover_datasets pattern="__designation_and_ext__"/>
+       <collection name="templates" type="list" label="Data table templates">
+             <discover_datasets pattern="(?P&lt;designation&gt;.+\.txt)" format="txt"/>
```

Comment:
Do you want to add a test here for this change?

Reply:
If ok for you we will see for further update of the tool and merge today this so I can try to test it on eu next week and share it to engineer we have who need to use it if possible